### PR TITLE
Merge virtual-merge in from dor-utils

### PR DESCRIPTION
**Pull request by jcoyne**
This will allow us to deprecate dor-utils and just maintain this script in Argo

Ref sul-dlss/dor-utils#66
Ref #1323
Ref #1278

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,17 @@ Dor.find("druid:pv820dk6668").destroy
 %w[pv820dk6668 rn653dy9317 xb482bw3979 hj185vb7593 hv992ry2431].each{ |pid| Dor.find("druid:#{pid}").destroy }
 ```
 
+### DOR virtual-merge
+
```
Comment (justinlittman) [inline README.md]: Can you put a 1 sentence explanation of virtual-merge here?

Comment (jcoyne) [reply README.md]: I don't really know.  I'm just porting from https://github.com/sul-dlss/dor-utils/

Comment (justinlittman) [reply README.md]: "A tool to merge multiple objects into a parent object via virtual object structure, namely the externalFile element in contentMetadata." (which is clear as mud, but better than nothing)

Comment (ndushay) [reply README.md]: I'm writing stuff in ticket now.  See last comment:  https://github.com/sul-dlss/dor-utils/issues/17

Are we sure this is the ONLY use of dor-utils?
Are we sure putting this functionality in argo is the way forward?  dor-utils is on lyberservices-prod box and may need access to stuff there ...

I'm not opposed to moving it, but ... should we move it NOW?

**Comment by ndushay** [comment]
decision was made to hold off on this until we fix the script in dor-utils.  I'm closing this for now. I will make a ticket in dor-utils to migrate the script elsewhere so we can kill that code base.  :-)

**Comment by ndushay** [comment]
Ticket about this:  https://github.com/sul-dlss/dor-utils/issues/65

**Comment by ndushay** [comment]
good point, mike!  I will bump dor-services, too!

**Pull request merged**